examples/README.md

```diff
@@ -150,7 +150,7 @@ To run it in each of these various modes, use the following commands:
     * With Accelerate config and launcher
         ```bash
         accelerate config  # This will create a config file on your server
```

Comment:
```suggestion
        accelerate config  # This will create a config file on your server at `~/.cache/huggingface/accelerate/default_config.yaml`
```